{"timestamp":"2025-07-10T14:25:15.909320","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-10T14:25:15.910195","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/bangalore_home_prices_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-10T14:25:15.912638","level":"info","event":"Using MLflow tracking URI: http://mlflow:5000","logger":"root"}
{"timestamp":"2025-07-10T14:25:18.099118","level":"info","event":"Starting model training...","logger":"root"}
{"timestamp":"2025-07-10T14:25:20.211110","level":"info","event":"Connection Retrieved 'aws_default'","logger":"airflow.hooks.base"}
{"timestamp":"2025-07-10T14:25:20.211273","level":"info","event":"AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.","logger":"airflow.providers.amazon.aws.utils.connection_wrapper.AwsConnectionWrapper"}
{"timestamp":"2025-07-10T14:25:21.658672","level":"info","event":"Preparing features...","logger":"root"}
{"timestamp":"2025-07-10T14:25:21.959773","level":"info","event":"Splitting data into train and test sets...","logger":"root"}
{"timestamp":"2025-07-10T14:25:21.966926","level":"info","event":"MLflow tracking URI set to: http://mlflow:5000","logger":"root"}
{"timestamp":"2025-07-10T14:25:22.021479","level":"warning","event":"Retrying (Retry(total=6, connect=6, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x7afb28170fe0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)\")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T14:25:26.581949","level":"warning","event":"Retrying (Retry(total=5, connect=5, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x7afb28078890>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)\")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T14:25:35.343612","level":"warning","event":"Retrying (Retry(total=4, connect=4, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x7afb28078500>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)\")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T14:25:51.599199","level":"warning","event":"Retrying (Retry(total=3, connect=3, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x7afb280787a0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)\")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T14:26:24.262467","level":"warning","event":"Retrying (Retry(total=2, connect=2, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x7afb28078dd0>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)\")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T14:27:28.837407","level":"warning","event":"Retrying (Retry(total=1, connect=1, read=7, redirect=7, status=7)) after connection broken by 'NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x7afb28078e90>: Failed to resolve 'mlflow' ([Errno -3] Temporary failure in name resolution)\")': /api/2.0/mlflow/experiments/get-by-name?experiment_name=bangalore_home_prices","logger":"urllib3.connectionpool"}
{"timestamp":"2025-07-10T14:27:48.938897Z","level":"error","event":"Server indicated the task shouldn't be running anymore. Terminating process","detail":{"detail":{"reason":"not_running","message":"TI is no longer in the running state and task should terminate","current_state":"failed"}},"logger":"task"}
{"timestamp":"2025-07-10T14:27:48.973276Z","level":"error","event":"Task killed!","logger":"task"}
